File: uva10035.cpp
#include <iostream>
using namespace std;

int main(){
    ios_base::sync_with_stdio(0);
    cin.tie(0);

    int a,b;
    while (cin>>a>>b)
    {
        if(a==0&&b==0){
            break;
        }

        int ans=0;
        int carry =0;
        while(a!=0||b!=0){
            int temp=a%10+b%10+carry;
            carry=temp/10;
            if(carry==1){
                ans+=1;
            }
            a=a/10;
            b=b/10;
           
        }

        // cout<<ans<<endl;

        if(ans==0){
            cout<<"No carry operation."<<endl;
        }
        else if(ans==1){

            cout<<"1 carry operation."<<endl;
        }
        else{
            cout<<ans<<" carry operations."<<endl;
        }
    }
    
}

File: uva10035.py
while True:
    a,b=map(str,input().split())
    if a=='0' and b=='0':
        break
    a=a[::-1]
    b=b[::-1]
    if len(b)>len(a):
        a,b=b,a
    
    carry=0
    ans=0
    for i in range(len(a)):
        temp=int(a[i])+carry
        if i<len(b):
            temp+=int(b[i])
        carry=temp//10
        ans+=carry
    
    if ans == 0:
        print("No carry operation.")
    elif ans == 1:
        print("1 carry operation.")
    else:
        print(f"{ans} carry operations.")
    
    
    
    
    
    
    
# AC

# while True:
#     a,b=map(int ,input().split())
#     if a==0 and b==0 :
#         break
#     ans=0
#     c=0
#     while a or b:
#         if a%10+b%10+c>9:
#             ans=ans+1
#             c=1
#         else:
#             c=0

#         a=a//10
#         b=b//10

#     if ans==0:
#         print("No carry operation.")
#     elif ans==1:
#         print(f'{ans} carry operation.')
#     else:
#         print(f'{ans} carry operations.')
    
    